cmd/manager/profileparser.go

```diff
@@ -142,6 +177,37 @@ func runProfileParser(cmd *cobra.Command, args []string) {
 
 	err = profileparser.ParseBundle(contentDom, pb, pcfg)
 
+	if err == nil {
+		// associate profile names with rules
+		profileRuleMap, err := fetchProfileRuleMap(pcfg)
+		if err != nil {
+			cmdLog.Error(err, "Couldn't fetch profile rule map")
```

Comment:
nit: The user reading this probably doesn't need to know the implementation details of how we're attempting to associate profiles to rules, which is using maps. What they might care about is why we couldn't list the profiles (and hopefully that's described in the `err` from `fetchProfileMap`), and that building the rules failed.

```go
cmdLog.Error(err, "Failed to associate profiles to rules due to an error fetching profiles") 
```

Additionally, what should they do in this case?